src/plugins/pinDms/data.ts

```diff
@@ -106,7 +106,12 @@ export function categoryLen() {
 }
 
 export function getAllUncollapsedChannels() {
-    return categories.filter(c => !c.collapsed).map(c => c.channels).flat();
+    return categories.filter(c => !c.collapsed).map(c => {
```

Comment:
i know this was already this way before your changes, but `.map(...).flat()` is better written as `.flatMap(...)`
```suggestion
    return categories.filter(c => !c.collapsed).flatMap(c => {
```